sleuth_crawler/scraper/scraper/spiders/parsers/generic_page_parser.py

```diff
@@ -6,13 +6,19 @@ def parse_generic_item(response, children):
     """
     Scrape generic page
     """
+    site_title = ""
     title = utils.extract_element(response.xpath("//title/text()"), 0)
+    titles = title.split('|')
+    if len(titles) == 2:
+        title = titles[0].strip()
+        site_title = titles[1].strip()
     desc = utils.extract_element(response.xpath("//meta[@name='description']/@content"), 0)
     raw_content = utils.strip_content(response.body)
```

Comment:
In the case where there is an error in `strip_content` it will return an array with an empty string, so we should probably check for that and not try to create a ScrapyGenericPage() in that case.

Reply:
@bfbachmann My goal with `strip_content` returning an empty string was to save non-webpage links, since PDFs are still somewhat relevant search results, though if you don't think that's a good idea I'll change it

Reply:
I am going to leave this in for now, since we might want to handle other file types properly in the future